pkg/admission/server_test.go

```diff
@@ -602,6 +602,262 @@ func TestServeHTTPSubmissions(t *testing.T) {
 					},
 				},
 			},
+			{
+				name: "v1a1 GatewayClass create events do not result in an error",
```

Comment:
Not a blocker on this PR, but it may make sense to split these tests out into different files or at least different sections per API version so that if/when we remove support for v1alpha1 it's easier to pull it out in one piece.

Reply:
I had the same thought. I've this on my todo.